components/servicios.tsx

```diff
@@ -0,0 +1,46 @@
+import Image from 'next/image'
+import SectionTitle from './ui/section-title'
+import Browser from '@/public/images/Browser.svg'
+import Rocket from '@/public/images/Rocket.svg'
+import Mobile from '@/public/images/Mobile.svg'
+
+export default function Servicios() {
+  return (
+    <section>
+      <div className="max-w-6xl mx-auto px-4 sm:px-6">
+        <div className="py-12 md:py-20">
+
+          <SectionTitle subtitle='Servicios' title='¿Qué hacemos?' />
+
+          <div className="max-w-sm mx-auto grid gap-8 lg:grid-cols-3 lg:gap-6 items-start lg:max-w-none pt-11">
+
+            <div className="flex flex-col h-full p-6 grow shrink basis-0 self-stretch bg-white bg-opacity-7 rounded-2xl border-2 border-blue-600 justify-start items-center gap-6" data-aos="fade-up">
+              <div>
+                <Image className="w-full h-11" src={Browser} width={48} height={48} alt="Web Browser" />
+              </div>
+              <div className="text-center text-2xl font-medium">Aplicaciones Web </div>
+              <div className="text-justify text-lg text-gray-400">Creadas para adaptarse a los flujos de trabajo de tu empresa e integrar sin esfuerzo tus operaciones existentes.</div>
```

Comment:
El gap entre el ícono y los textos es gap-6 (lo tienes bien) pero entre el título y descripción es menor. Debería ser gap-2.5 pero dejémoslo mejor con gap-3. Habría que hacer otro div con flex para que tenga ese gap. Si no el espacio se siente mucho entre los textos. 

```suggestion
              <div className="flex flex-col gap-3">
                <div className="text-center text-2xl font-medium">Aplicaciones Web </div>
                <div className="text-justify text-lg text-gray-400">Creadas para adaptarse a los flujos de trabajo de tu empresa e integrar sin esfuerzo tus operaciones existentes.</div>
              </div>
```

Reply:
Quedó mejor así, gracias.